scrape/target.go

```diff
@@ -264,6 +274,30 @@ func (t *Target) LastScrapeDuration() time.Duration {
 	return t.lastScrapeDuration
 }
 
+// PreviousError returns the error encountered during the last errorful scrape.
```

Comment:
The mix of last and previous here is confusing.

Reply:
Yes but the v1 HTTP API (used by dashboards and UIs) is considered stable, so I have to keep last in the JSON API.

Maybe I can change to `current` or `lastCompleted` in the go variables?

Reply:
The Go should follow the JSON, or that'd also be confusing.